### DISCUSSION TITLE
How to set the format of loading other slices

### DISCUSSION THREAD

**Discussion by lc707776289** [mapsforge/vtm#1042] [Q&A] [answered]
Multiple slice sources are supported. How should I switch to mapbox slices on Android?

**Comment by devemux86** [comment, marked as answer]
There should be support for offline .mbtiles, see the [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).
There doesn't seem to be implemented (?) a decoder for Mapbox online vector tiles.

There is support for online vector tiles (OpenMapTiles format), see the `OpenMapTilesMvtActivity`, `MapilionMvtActivity`.

**Reply by lc707776289** [reply]
I didn't see how to switch between those two activities. My question is that the slice loaded on the current map is WGS84. How can I switch to the slice of the Mercator coordinate system?

**Reply by devemux86** [reply]
[Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) is the standard in mapping applications.
Geographic coordinates in API are defined in [WGS84](https://en.wikipedia.org/wiki/WGS84).

**Reply by lc707776289** [reply]
The default source of vtm tiles is Web Mercator or Mercator Projection. If it is Web Mercator, can I use Mercator Projection? On the contrary, if it is Mercator Projection, can I use Web Mercator?

**Reply by lc707776289** [reply]
What if it's a UTM projection?

**Reply by devemux86** [reply]
For the projections you can check the documentation available online.
https://en.wikipedia.org/wiki/Web_Mercator_projection

For the use of VTM please study the samples in [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).
